vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -517,6 +510,25 @@
     name: alfresco
     state: started
 
+- name: install bcrypt for hashing
+  become: yes
+  pip:
```

Comment:
Do you have to specify pip3 somehow here?  On some sites if pip2 is installed, I've seen an error when running plain pip, without specifying the pip version.